modules/040-control-plane-manager/monitoring/prometheus-rules/etcd-maintenance.yaml

```diff
@@ -0,0 +1,60 @@
+- name: d8.etcd-maintenance.quota-backend-bytes
+  rules:
+    - alert: D8KubeEtcdDatabaseSizeCloseToTheLimit
+      expr: max by (node) (etcd_mvcc_db_total_size_in_bytes{job="kube-etcd3"} >= max(d8_etcd_quota_backend_total)) * 0.9
+      labels:
+        severity_level: "6"
+        tier: cluster
+      annotations:
+        plk_protocol_version: "1"
+        plk_markup_format: "markdown"
+        plk_pending_until_firing_for: "10m"
```

Comment:
don't you want to move it to `for: 10m` instead of polk annotation?

Reply:
I don't know :-) This alert was moved from [monitoring-kubernetes-control-plane module](https://github.com/deckhouse/deckhouse/pull/1389/files/ff50c53fa2a0f8c9a9b1b5df058f7ecc43a3e021#diff-490f6a44424e715726f88cfda3b952a536093e8e93da155fa1be8ffe0ea57590L118) with change in expression.

Reply:
@nabokihms What do you thing about it?

Reply:
Yes, you should use `for`. We are making such changes for other alerts in https://github.com/deckhouse/deckhouse/pull/1446

Reply:
done